src/vinyldns/boto_request_signer.py

```diff
@@ -84,6 +84,12 @@ def build_auth_header(self, method, path, headers, body, params=None):
         credential_scope = u'/'.join([request.timestamp, request.region_name, request.service_name, u'aws4_request'])
 
         canonical_request = auth_handler.canonical_request(request)
+        split_request = canonical_request.split('\n')
+
+        if params != {} and split_request[2] == '':
```

Comment:
by default Boto removes POST request query params. It's their position that those params should be in the body of the request. https://github.com/boto/boto/blob/develop/boto/auth.py#L393-L403. This makes the signature it generates and the signature the API is expecting different when the `allowManualReview` query parameter is passed in the `create_batch_change` POST request.

To get around this we check if there are parameters but the parameter part of the canonical request string is empty. If that's the case we add the params to the query.


As a note, this wasn't an issue when using Postman, but I don't know if this is going to be an issue with any of the other VinylDNS clients.